### ISSUE TITLE
Allows attach to "wrong" URL.

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#252]
**Reported by Rattledagger on 26 Mar 37444223 22:13 UTC**
None

**Comment by romw** [comment]
**Commented by Didactylos on 7 Mar 37879969 17:46 UTC**
Without further explanation, this should be closed.

**Comment by romw** [comment]
**Commented by Rattledagger on 18 Feb 37880153 05:46 UTC**
If someone types-in a valid alias for a projects URL, the BOINC-client will accept this, but for every scheduler-URL they'll get an error-message.


```
17.11.2007 18:08:09||Fetching configuration file from http://boinc.berkeley.edu/alpha/get_project_config.php
17.11.2007 18:08:25|BOINC alpha test|[Starting scheduler request
17.11.2007 18:08:25|BOINC alpha test|[sched_op_debug](sched_op_debug]) Fetching master file
17.11.2007 18:08:30|BOINC alpha test|[Got master file; parsing
17.11.2007 18:08:30|BOINC alpha test|[sched_op_debug](sched_op_debug]) Found 1 scheduler URLs in master file
17.11.2007 18:08:30|BOINC alpha test|Master file download succeeded
17.11.2007 18:08:35|BOINC alpha test|[Starting scheduler request
17.11.2007 18:08:36|BOINC alpha test|Sending scheduler request: Project initialization.  Requesting 1 seconds of work, reporting 0 completed tasks
17.11.2007 18:08:41|BOINC alpha test|Scheduler request succeeded: got 0 new tasks
17.11.2007 18:08:41|BOINC alpha test|[sched_ops_debug](sched_op_debug]) Server version 601
17.11.2007 18:08:41|BOINC alpha test|You used the wrong URL for this project
17.11.2007 18:08:41|BOINC alpha test|The correct URL is http://isaac.ssl.berkeley.edu/alpha/
17.11.2007 18:08:41|BOINC alpha test|Using the wrong URL can cause problems in some cases.
17.11.2007 18:08:41|BOINC alpha test|When convenient, detach this project, then reattach to http://isaac.ssl.berkeley.edu/alpha/
```

**Comment by romw** [comment]
**Commented by romw on 22 Oct 39178408 09:20 UTC**
(In [17513])     - MGR: Newer versions of the server-side software publish their
        master url in the response to the get_project_config.php
        request.  If it exists use it over what was specified
        by the user.  Fixes #259
        
    clientgui/
        AccountManagerProcessingPage.cpp
        ProjectProcessingPage.cpp

**Comment by romw** [comment]
**Commented by romw on 29 Jul 39180811 23:33 UTC**
(In [17534])     - MGR: Newer versions of the server-side software publish their
        master url in the response to the get_project_config.php
        request.  If it exists use it over what was specified
        by the user.  Fixes #259
        
    clientgui/
        AccountManagerProcessingPage.cpp
        ProjectProcessingPage.cpp
